Library/Formula/cryptol.rb

```diff
@@ -0,0 +1,53 @@
+require "language/haskell"
+
+class Cryptol < Formula
+  include Language::Haskell::Cabal
+
+  homepage "http://www.cryptol.net/"
+  url "https://github.com/GaloisInc/cryptol/archive/v2.1.0.tar.gz"
+  sha1 "b0432097c807518eafe138c77a02d8142cecc223"
+  head "https://github.com/GaloisInc/cryptol.git"
+
+  depends_on "ghc" => :build
+  depends_on "cabal-install" => :build
+  depends_on "cvc4"
+
+  # patch to allow parallelized builds
+  patch :DATA
+
+  def install
+    cabal_sandbox do
+      system "./configure"
+      system "make", "PKG=#{prefix}", prefix
```

Comment:
Reported upstream: https://github.com/GaloisInc/cryptol/issues/127

Future releases will likely use `share/cryptol` rather than `lib` for the built-in libraries.